crates/fuel-core/src/executor.rs

```diff
@@ -1500,7 +1500,7 @@ where
                     backtrace.contract(),
                     backtrace.registers(),
                     backtrace.call_stack(),
-                    hex::encode(&backtrace.memory()[..backtrace.registers()[RegId::SP] as usize]), // print stack
+                    hex::encode(&backtrace.memory()[..usize::try_from(backtrace.registers()[RegId::SP]).expect("`$sp` can't be more than memory - `usize`")]), // print stack
```

Comment:
This `expect` statement is confusing me. 

> memory - `usize`

Are we subtracting the `usize`? Do you mean to say

> memory (`usize`)

or

> memory, which is a `usize`

or something like that?